addon/components/lt-infinity.js

```diff
@@ -1,38 +1,39 @@
 import Component from '@ember/component';
 import layout from '../templates/components/lt-infinity';
-import InViewportMixin from 'ember-in-viewport';
 import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
 
-export default Component.extend(InViewportMixin, {
-  classNames: ['lt-infinity'],
-  classNameBindings: ['viewportEntered:in-viewport'],
+export default Component.extend({
   layout,
 
-  rows: null,
+  inViewport: service(),
```

Comment:
`inViewportService: service('in-viewport')`

Reply:
See https://github.com/offirgolan/ember-light-table/pull/693#issuecomment-496007324